src/map/atcommand.c

```diff
@@ -4123,16 +4123,36 @@ ACMD(mapinfo)
 			for (i = 0; i < map->list[m_id].npc_num;) {
 				struct npc_data *nd = map->list[m_id].npc[i];
 				switch(nd->dir) {
```

Comment:
good catch, will look into it.

Reply:
Please move this to a different issue, as I will not cover this with this PR as I want to focus on unit.c

Reply:
That's not worth creating an issue, since it's just a small code improvement.
If you don't want to change it, just leave it as it is and resolve this conversation. :-)